src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -685,10 +685,12 @@ private function addSessionSection(ArrayNodeDefinition $rootNode): void
                         ->integerNode('sid_length')
                             ->min(22)
                             ->max(256)
+                            ->setDeprecated('symfony/framework-bundle', '7.2', 'Setting the "%path%.%node%" configuration option is deprecated. It will be removed in version 8.0.')
```

Comment:
For deprecations, it's advised to always provide an alternative solution. In this case, there's no alternative because PHP itself is deprecating it. But, I wonder if we should mention that too.

E.g.:

```
Setting the "%path%.%node%" configuration option is deprecated. It will be removed in version 8.0.

Setting the "%path%.%node%" configuration option is deprecated and it will be removed in version 8.0. There is no alternative available, as PHP 8.4 has deprecated the related option.
```

Reply:
I agree, it's always a bit harsh to be faced with a depreciation that gives no alternative and that's it (even if there isn't one).